lib/booktype/apps/core/static/core/js/booki.js

```diff
@@ -760,15 +760,16 @@
       */
       notification: function (title, message, settings) {
         var opts = win.booktype.editor.data.settings.config.notifications.settings;
+        var icon = 'icon-bullhorn';
 
         jquery.notify({
-            icon: settings.icon || 'icon-bullhorn',
```

Comment:
Hi @eos87 , 
Yes, I agree with you. The issue is that at the moment `settings.icon` produces an error, it's undefined. Because of this notifications don't work. I've already created BK-2221 regarding this issue. Removing of `settings.icon` it's just a quick fix. I will do the proper fix asap and for sure `settings.icon` will be with us :kick_scooter:  
Here is a link to all notification settings [I'm link](https://github.com/sourcefabric/Booktype/commit/bfec52f3fdedb08e6fddbea979aa2ba814b85152#diff-63ebc0e85015c9c06865b0d69e9a263aR86)